packages/design-system-tailwind-preset/src/typography.ts

```diff
@@ -20,13 +20,7 @@ export const typography = {
     'l-body-xs': 'var(--typography-l-body-xs-font-size)',
   },
   fontFamily: {
-    sans: [
-      'Euclid Circular B', // This should match @font-face set in tailwind.css
-      'Helvetica Neue',
-      'Helvetica',
-      'Arial',
-      'sans-serif',
-    ],
+    sans: ['var(--font-family-sans)'],
```

Comment:
Updating to point to the new `--font-family-sans` CSS variable, this will prevent us having to update the preset if we need to change what `--font-family-sans` points to